fork-common/src/main/java/com/shazam/fork/system/axmlparser/ApplicationInfoFactory.java

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Crazy idea but wouldn't aapt -badging give us similar info?

Reply:
yeah, I'll try to dig into it. If it's useful I'll open another PR!

Reply:
👍